checkbox-ng/checkbox_ng/launcher/subcommands.py

```diff
@@ -1340,31 +1371,47 @@ def invoked(self, ctx):
         tp = self.sa._context._test_plan_list[0]
         tp_us = TestPlanUnitSupport(tp)
         self.override_list = tp_us.override_list
+
         jobs_and_templates_list = select_units(
             all_jobs_and_templates,
             [tp.get_mandatory_qualifier()] + [tp.get_qualifier()],
         )
+        relevant_manifest_units = self._get_relevant_manifest_units(
+            jobs_and_templates_list
+        )
 
+        units_to_print = itertools.chain(
+            relevant_manifest_units, iter(jobs_and_templates_list)
+        )
         obj_list = []
-        for unit in jobs_and_templates_list:
+        for unit in units_to_print:
             obj = unit._raw_data.copy()
             obj["unit"] = unit.unit
             obj["id"] = unit.id  # To get the fully qualified id
-            obj["certification-status"] = (
-                self.get_effective_certification_status(unit)
-            )
-            if unit.template_id:
```

Comment:
Did you remove this condition intentionally? While playing with your branch, I compared the JSON output between `main` and your branch and noticed a lot of `template_id: null` in there.

Reply:
no I've made a mistake, the check for none template_id is still needed